src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {

```

Comment:
```suggestion
public class FigureSupplier {
```
if you want to use ColorSupplier, create an instance of it as a global field in this class